index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>MY PORTFOLIO</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
+  </head>
+  <body>
+    <header class="hero">
+      <nav class="herocontent">
+        <a href="#" class="logo">Welcome</a>
+        <ul class = "nav-list">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+        <label for="check" class="hambuger-btn">
+          <i class = "fas fa-bars"></i>
+        </label>
+      </nav>
+    </header>
+    <section>
+      <div class="content">
+        <h1>Hey there. I'm Bobson</h1>
+        <h2>I'm a software developer and an author</h2>
+        <P class = "supporting-text">I can help you build a feature, product or a website.
+        Look through some of my work and experience!
+        If you like what you see and have a project you need coded, dont
+          hesitate to contact me.
+        </p>
+      </div>
+      <section>
+      <ul class="socialmedia">
+        <li>
+          <a href="https://twitter.com/KALUNGIBOBSONJ">
+              <img src="./images/socialmedia icons/twitter.png" alt="twitter icon">
+          </a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/KALUNGI-J-BOBSON">
+              <img src="./images/socialmedia icons/linkedin.png" alt="linkedin icon">
+          </a>
+        </li>
+        <li>
+          <a href="https://medium.com/@kalungijbobson">
+              <img src="./images/socialmedia icons/mail.png" alt="Medium icon" />
+          </a>
+      </li>
+      <li>
+      <a href="https://github.com/KALUNGI-J-BOBSON">
+        <img src="./images/socialmedia icons/github.png" alt="github icon">
+    </a>
+   </li>
+   <a href="#">
+    <img src="./images/socialmedia icons/angelist 1.png" alt="angelist icon">
+</a>
+</li>
+    </ul>
+  </section>
+      <section class="disabled">
+        <img
+          src="./images/Disabled.png"
+          alt="disabled vector"
+        />
+      </section>
+      
```

Comment:
- Hey, great with your layout, however, you've nested other sections in this introductory section of your page and you didn't add the closing section tag also as this is not the ideal way to use the semantic tag. This how the layout is meant to be 

```

<section>
content ........
content......
content......
<ul>
<li>.........</li>
</ul>
<div>
<img/>
</div>
</section>
```

Please, let's kindly fix this to improve our layout and make our markup readable. 

---

Please, let's also add the missing background image of this section to match the design as much as we can to improve our design.